config/repos.groups.yml

```diff
@@ -0,0 +1,29 @@
+---
+providers: &providers
+  manageiq-providers-amazon:
+  manageiq-providers-ansible_tower:
+  manageiq-providers-autosde:
+  manageiq-providers-azure:
+  manageiq-providers-azure_stack:
+  manageiq-providers-foreman:
+  manageiq-providers-google:
+  manageiq-providers-ibm_cloud:
+  manageiq-providers-ibm_terraform:
+  manageiq-providers-kubernetes:
+  manageiq-providers-kubevirt:
+  manageiq-providers-lenovo:
+  manageiq-providers-nsxt:
+  manageiq-providers-nuage:
+  manageiq-providers-openshift:
+  manageiq-providers-openstack:
+  manageiq-providers-ovirt:
+  manageiq-providers-redfish:
+  manageiq-providers-scvmm:
+  manageiq-providers-vmware:
+core: &core
+  manageiq:
+  manageiq-api:
+  manageiq-ui-classic:
```

Comment:
Can you please also add most of the following?

```yaml
  manageiq-automation_engine:
  manageiq-consumption:
  manageiq-content:
  manageiq-decorators:
  manageiq-gems-pending:
  manageiq-graphql:
  manageiq-schema:
  manageiq-ui-service:
  manageiq-v2v:
```

Although ui-service seems to fail, so maybe we should not put that one?